.github/workflows/verify-older-os-compatibility.yml

```diff
@@ -19,46 +19,38 @@ jobs:
       matrix:
         include:
         
+        - version: '15.0'
```

Comment:
Xcode15 is not enough to submit, adding it is redundant. Maybe we can change this job to test for beta xcodes?